client/my-sites/theme/main.jsx

```diff
@@ -345,33 +346,53 @@ const ThemeSheet = React.createClass( {
 		const analyticsPath = `/theme/:slug${ section ? '/' + section : '' }${ siteID ? '/:site_id' : '' }`;
 		const analyticsPageTitle = `Themes > Details Sheet${ section ? ' > ' + titlecase( section ) : '' }${ siteID ? ' > Site' : '' }`;
 
+		const Head = this.props.isLoggedIn
+			? require( 'layout/head' )
+			: require( 'my-sites/themes/head' );
```

Comment:
I'd really love if we could get rid of the conditional import while we're here; I'm quite sure `layout/head` is all we ever need here. You can read its source (and compare to what fields we override here in `main.jsx`) to validate that assumption -- or maybe just try without and compare resulting `title` and `meta`s. :-)